apps/expo/.env.development

```diff
@@ -1,30 +1,31 @@
-STAGE=development
-NEXT_PUBLIC_CHAIN_ID=mumbai
+STAGE=production
```

Comment:
😄

Reply:
sorry, it's for test, i have changed back in my local, will push it soon.